### PR TITLE
Updates based on 21CR6

### PR DESCRIPTION
**Pull request by cverges-medallia**

### CODE REVIEW

File: speech.yaml [hunk 1]
```diff
@@ -226,32 +238,75 @@ components:
...
+            metadata available in `speech_additional_info`.  This field was
+            introduced with Medallia Experience Cloud 22CR3.0.
+        speech_additional_info:
+          type: object
+          description:
+            Additional metadata to add to the Medallia Speech record.  Mapping this
+            data to the record is performed in setting up the Medallia Speech API
+            Connector in Admin Suite.  Use of this field requires setting a
+            `connector_id`.
+          additionalProperties:
```
Comment (jmarshall-medallia) [inline speech.yaml:302]: We should be able to remove this additionalProperties field as `speech_additional_info` is already type object and supports any type of valid json schema (nested objects, array, all types, etc)

File: speech.yaml [hunk 2]
```diff
@@ -226,32 +238,75 @@ components:
...
           description:
-            The unique identifier (UUID) of the customer profile
-            (will be deprecated soon, use `connection_id` instead)
+            This field has been deprecated since Medallia Experience Cloud
+            version 21CR3.0, please use `connection_id` instead.
           maxLength: 10240
+        engine:
+          type: string
+          description:
+            The Medallia Speech engine to use to perform the
```
Comment (jmarshall-medallia) [inline speech.yaml:277]: Maybe here its worth mentioning that if you define an engine, there will be a locale+engine combo validation in the API.

Something like this but better worded: "If using engine parameter, please validate with your Medallia speech specialist which locales are supported for each engine" or "ask for recommended engine + locale combos"

**Review by jmarshall-medallia — APPROVED** [review]
lgtm!

File: speech.yaml (hunk 2 above)
Comment (cverges-medallia) [reply speech.yaml:277]: Does the existing line at the end of the description cover?

> The value to use is determined by your Medallia Speech specialist during the initial audio analysis process while onboarding to Medallia Speech.

Comment (jmarshall-medallia) [reply speech.yaml:277]: Yes, pretty much. 
Just wanted to be clear that a request like this:

locale: el  (greek)

can be accepted but if you also add an engine (which doesn't support that locale)

locale: el
engine: Engine 1

will be rejected.

**Pull request merged**